stage1/00-boot-files/files/config.txt

```diff
@@ -59,7 +59,7 @@ camera_auto_detect=1
 display_auto_detect=1
 
 # Enable DRM VC4 V3D driver
-dtoverlay=vc4-kms-v3d
+#dtoverlay=vc4-kms-v3d
```

Comment:
[    8.882966] vc4-drm gpu: failed to bind fe400000.hvs (ops vc4_hvs_ops [vc4]): -2
[    8.883187] vc4-drm gpu: master bind failed: -2
[    8.883257] vc4-drm: probe of gpu failed with error -2
[    9.808908] v3d: probe of fec00000.v3d failed with error -2